Source/WebKit/UIProcess/Extensions/WebExtension.h

```diff
@@ -63,6 +64,8 @@ class WebExtension : public API::ObjectImpl<API::Object::Type::WebExtension>, pu
     WTF_MAKE_NONCOPYABLE(WebExtension);
 
 public:
+    using IconVariant = std::variant<RefPtr<WebCore::Icon>, Vector<double>>;
```

Comment:
`IconVariant ` is confusing, since this isn't about extension icon variants but `std::variant`.
```suggestion
    using IconCacheEntry = std::variant<RefPtr<WebCore::Icon>, Vector<double>>;
```